idsa/.htaccess

```
@@ -1,7 +1,13 @@
 Options +FollowSymLinks
 RewriteEngine on
-RewriteRule ^$ https://www.internationaldataspaces.org/ [R=302]
-RewriteRule ^(core|code)($|/(.*?$)) https://github.com/IndustrialDataSpace/InformationModel/releases/download/v1.0.0/IDS-InformationModel_v1.0.0.ttl [R=302]
+#RewriteRule ^$ https://www.internationaldataspaces.org/ [R=302]
+RewriteRule ^(core|code)($|/$) https://industrialdataspace.github.io/InformationModel/ [R=302]
```

I think you could simplify to:
```suggestion
RewriteRule ^(core|code)/?$ https://industrialdataspace.github.io/InformationModel/ [R=302]
```

---

I updated the file accordingly.